### PR TITLE
add missing docs for Deploy Designs

### LINKED ISSUE #344: [Docs] Deploying Designs
#### Current State
This page is missing content - https://docs.layer5.io/meshmap/tasks/designs/deploying-designs/

#### Desired State
[Mastering Meshery Learning Path](https://meshery.layer5.io/academy/learning-paths/mastering-meshery) content or Meshery Docs content can be reused (copied and pasted) here.

---

#### Contributor Guide and Resources
- 📚 [Instructions for contributing to documentation](https://github.com/layer5io/docs/blob/master/CONTRIBUTING.md)
   - Layer5 documentation [site](https://docs.layer5.io) and [source](https://github.com/layer5io/docs/)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sparshagl**
**Notes for Reviewers**

This PR fixes #344

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by sparshagl** [comment]
@leecalcote @Alero-Awani

**Comment by Alero-Awani** [comment]
@sparshagl Thank you for this. I am creating new sections in the layer docs. I will assign one of those to you.

File: content/en/meshmap/getting-started/images/deploy-designs/action.png
Comment (marblom007) [inline content/en/meshmap/getting-started/images/deploy-designs/action.png:1]: This is a nice distinction.

File: content/en/meshmap/tasks/designs/deploying-designs.md [hunk 1]
```diff
@@ -7,10 +7,58 @@ categories: [Designer]
...
+
+The Notification Center in Meshery helps manage events during the deployment process. It provides real-time updates and alerts on the status of the deployment. This feature can be particularly useful for troubleshooting, as it:
+
+1. Displays immediate feedback on the success or failure of each deployment step.
+
+2. Highlights specific error messages and codes, helping you quickly identify and understand issues.
+
+3. Offers links to detailed documentation and guides for resolving common problems.
+
+4. Keeps a log of past notifications, allowing you to track and review previous errors and their resolutions.
```
Comment (marblom007) [inline content/en/meshmap/tasks/designs/deploying-designs.md:55]:
```suggestion
1. Displays immediate feedback on the success or failure of each deployment step.
1. Highlights specific error messages and codes, helping you quickly identify and understand issues.
1. Offers links to detailed documentation and guides for resolving common problems.
1. Keeps a log of past notifications, allowing you to track and review previous errors and their resolutions.
```

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thanks, @sparshagl!

**Pull request merged**